app/Http/Controllers/LandingPagesController.php

```diff
@@ -93,6 +93,15 @@ public function show($id, $slug = null)
             'intro' => $item->visit_hour_intro
         ];
 
+        $faqs = [];
+
+        foreach ($item->faqs as $faq) {
+            array_push($faqs, [
+                'question' => $faq->question,
+                'answer' => $faq->answer,
+            ]);
+        }
+
```

Comment:
Because the Faq model really only has these two columns, you could remove this and pass the entire collection down below:

```
'faqs' => $item->faqs->all(),
```

Then in the `_footer----visit.blade.php` the syntax of the `@foreach` would stay the same, but you would reference these values as as object instead of an array. e.g., `$faq->question` instead of `$faq['question']`